packages/menu/src/styles.tsx

```diff
@@ -0,0 +1,96 @@
+import { css } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+
+const indentation = 15;
+const leftBar = 5;
+
+export const menuItemContainerStyle = css`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding-left: ${indentation}px;
+  padding-right: ${indentation}px;
+  text-decoration: none;
+  cursor: pointer;
+  text-decoration: none;
+  position: relative;
+  transition: background-color 150ms ease-in-out;
+  border: none;
+  margin: unset;
+  width: 100%;
+  font-family: Akzidenz, ‘Helvetica Neue’, Helvetica, Arial, sans-serif;
+  box-sizing: border-box;
+  background: ${uiColors.white};
+  text-align: left;
+  &:focus {
+    outline: none;
+  }
+  &:before {
```

Comment:
Let's add line breaks before all pseudo-classes and states that are preceded by style declarations.

Reply:
👍